src/test/kotlin/se/uulm/snowballr/backend/validation/ProjectValidatorTest.kt

```diff
@@ -327,4 +327,45 @@ class ProjectValidatorTest {
             assertInvalidResult<BlankField>(result)
         }
     }
+
+    inner class InformationGetRequest {
```

Comment:
`@Nested` is required here; otherwise, the calls are not discovered by JUnit.
```suggestion
    @Nested
    inner class InformationGetRequest {
```